jquery.flot.composeImages.js

```diff
@@ -0,0 +1,177 @@
+(function($) {
+    "use strict";
+    const CGENERALFAILURECALLBACKERROR = -100; //simply a negative number
+    const CSUCCESSFULIMAGEPREPARATION = 0;
+    const CEMPTYARRAYOFIMAGESOURCES = -1;
+    const CNEGATIVEIMAGESIZE = -2;
```

Comment:
Since these variables are visible to this file only the C prefix convention is not needed.

Reply:
ok, that's fine. C stands for **C**onstant.